nrf_modem/doc/gnss_interface.rst

```diff
@@ -90,7 +90,7 @@ The data to be deleted is selected using a bitmap.
 
 .. note::
 
-   TCXO offset data is not obtained from satellites or A-GPS assistance.
+   TCXO offset data is not obtained from satellites or A-GNSS assistance.
```

Comment:
Technically the current sentence is not correct since the TCXO offset is estimated from the fix, that is, from the satellites. Proposal to replace line 93-95.
```suggestion
   TCXO offset is a characteristic of each device that changes slowly over time. Therefore, it remains usable for extended time periods and should typically not be deleted when simulating a cold start.
```

Reply:
Updated as discussed.